docs/getting-started/install-harper.md

```diff
@@ -0,0 +1,123 @@
+
+{% hint style="info" %}
+Test your Harper instance is up and running by querying `curl http://localhost:9925/health`
+{% endhint %}
+
+### Example Deployments
+To run a Harper container in the background with persistent storage and exposed ports, you can use a command like this:
+
+```bash
+docker run -d \
+  -v <host_directory>:/home/harperdb/hdb \
+  -e HDB_ADMIN_USERNAME=HDB_ADMIN \
+  -e HDB_ADMIN_PASSWORD=password \
+  -e THREADS=4 \
+  -p 9925:9925 \
+  -p 9926:9926 \
+  harperdb/harperdb
+```
+
+Here, the `<host_directory>` should be replaced with an actual directory path on your system where you want to store the persistent data. This command also exposes both the Harper Operations API (port 9925) and an additional HTTP port (9926).
+For a more advanced setup, enabling HTTPS and clustering, you can run:
+
+```bash
+docker run -d \
+  -v <host_directory>:/home/harperdb/hdb \
+  -e HDB_ADMIN_USERNAME=HDB_ADMIN \
+  -e HDB_ADMIN_PASSWORD=password \
+  -e THREADS=4 \
+  -e OPERATIONSAPI_NETWORK_PORT=null \
+  -e OPERATIONSAPI_NETWORK_SECUREPORT=9925 \
+  -e HTTP_SECUREPORT=9926 \
+  -e CLUSTERING_ENABLED=true \
+  -e CLUSTERING_USER=cluster_user \
+  -e CLUSTERING_PASSWORD=password \
+  -e CLUSTERING_NODENAME=hdb1 \
+  -p 9925:9925 \
+  -p 9926:9926 \
+  -p 9932:9932 \
+  harperdb/harperdb
+```
+
+In this setup, additional environment variables disable the unsecure Operations API port and enable secure ports for HTTPS, along with clustering parameters such as the clustering user, password, and node name. The port 9932 is also exposed for Harper clustering communication.
+
+Finally, if you simply wish to check the Harper version using the container, execute:
+
+```bash
+docker run --rm harperdb/harperdb /bin/bash -c "harperdb version"
+```
+
+This command runs the container momentarily to print the version information, then removes the container automatically when finished.
+
+### Logs and Troubleshooting
+To verify that the container is running properly, you can check your running containers with:
+
+```bash
+docker ps
+```
+
+If you want to inspect the logs to ensure that Harper has started correctly, use this command (be sure to replace `<container_id>` with the actual ID from the previous command):
+
+```bash
+docker logs <container_id>
+```
+
+Once verified, you can access your Harper instance by opening your web browser and navigating to http://localhost:9925 (or the appropriate port based on your configuration).
+
+### Offline Install
+If you need to install Harper on a device that doesn't have an Internet connection, you can choose your version and download the npm package and install it directly (you’ll still need Node.js and NPM). Click [this link](https://products-harperdb-io.s3.us-east-2.amazonaws.com/index.html) to download and install the package. Once you’ve downloaded the .tgz file, run the following command from the directory where you’ve placed it:
+
+```bash
+npm install -g harperdb-X.X.X.tgz harperdb install
+```
```

Comment:
I've never understood this - you still very much need an internet connection to download the package the first time 😆 . Is it better to say like "Raw binary installation" or something like that?